projects/memapCore/src/main/java/memap/components/prototypes/Storage.java

```diff
@@ -34,7 +37,10 @@ public Storage(String name, double capacity, double stateOfCharge, double max_ch
 		this.max_discharging = max_discharging;
 		this.effIN = effIN;
 		this.effOUT = effOUT;
-		this.storageLoss = storageLoss;
+		// TODO remove hard coding.
+		// this.storageLoss = storageLoss;
+		
+		this.storageLoss = 0.1;
```

Comment:
This should be removed and changed back to this.storageLoss = storageLoss as Storage.java is an abstract class. Values are set in the inheriting class when instantiated.